content/docs/command-reference/import-url.md

```diff
@@ -88,6 +88,15 @@ Specific explanations:
   running. DVC automatically expands this URL into a regular S3, SSH, GS, etc
   URL by appending `/path/to/file` to the `myremote`'s configured base path.
 
+The `deps` field in the generated `.dvc` file contains a hash value along with
+the `path` to track changes in the remote file/directory. The hash type depends
+on the type of remote location (protocol) being tracked. For instance,
+files/directories tracked from HTTP(s)/S3/Azure/GS remote locations will have an
```

Comment:
I made a mistake with the comment. GS uses `md5`. Sorry about that.

Reply:
Okay, I'll change it in the DVC Files and Directories page in the next commit :v:

Reply:
No worries @skshetry, glad you caught it though! Thanks for the followup. I updated #1469 and fixed the dvc.lock doc in another PR. But this brings some questions... I'll ping you somewhere else.

@utkarshsingh99 please update the paragraph above accordingly...